augly/image/intensity.py

```diff
@@ -100,6 +100,20 @@ def crop_intensity(metadata: Dict[str, Any], **kwargs) -> float:
     return resize_intensity_helper(metadata)
 
 
+def distort_barrel_intensity(a: float, b: float, c: float, d: float, **kwargs) -> float:
+    barrel_coefficients_magnitude = abs(a) + abs(b) + abs(c)
+    scale = np.abs(d)
+    intensity = 100 * (barrel_coefficients_magnitude / scale)
+    return float(np.clip(intensity, 0, 100))
```

Comment:
nit: can you make a helper function for these 4 lines of code since they're the same for these 2 intensity functions?